graphql-dgs-codegen-core/src/test/kotlin/com/netflix/graphql/dgs/codegen/KotlinCodeGenTest.kt

```diff
@@ -1685,29 +1686,20 @@ class KotlinCodeGenTest {
                 language = Language.KOTLIN
             )
         ).generate().kotlinDataTypes
-        assertThat(dataTypes[0].toString()).isEqualTo(
-            """
-                |package com.netflix.graphql.dgs.codegen.tests.generated.types
-                |
-                |import com.fasterxml.jackson.`annotation`.JsonProperty
-                |import kotlin.Deprecated
-                |import kotlin.ReplaceWith
-                |import kotlin.String
-                |
-                |@Deprecated(message = "This is going bye bye")
-                |public data class Person(
-                |  @JsonProperty("name")
-                |  @Deprecated(
-                |    message = "This field is no longer available",
-                |    replaceWith = ReplaceWith("firstName")
-                |  )
-                |  public val name: String? = null
-                |) {
-                |  public companion object
-                |}
 
-        """.trimMargin()
-        )
+        assertThat(dataTypes).hasSize(1)
+        assertThat(dataTypes[0].name).isEqualTo("Person")
+
+        val annotationSpec = (((dataTypes as ArrayList<*>)[0] as FileSpec).members[0] as TypeSpec).annotationSpecs[0]
+        assertThat((annotationSpec.typeName as ClassName).canonicalName).isEqualTo("kotlin.Deprecated")
+        assertThat(annotationSpec.members).hasSize(1)
+        assertThat(annotationSpec.members[0]).extracting("formatParts", "args").asList().contains(listOf("message = ", "%S"), listOf("This is going bye bye"))
+
+        val parameterSpec = (((dataTypes[0].members)[0] as TypeSpec).primaryConstructor as FunSpec).parameters[0]
+        assertThat(parameterSpec.name).isEqualTo("name")
+        assertThat(parameterSpec.annotations).hasSize(2)
+        assertThat((parameterSpec.annotations[0].typeName as ClassName).canonicalName).isEqualTo("com.fasterxml.jackson.annotation.JsonProperty")
```

Comment:
Assert for the argument in the `JsonProperty`.

Reply:
I think this would apply for all of the assertions for `JsonProperty` made below.

Reply:
So the json property one was not added by us. It was something that already existed.